test_suite/generic/test_generic.py

```diff
@@ -254,6 +254,24 @@ def test_grub_config(self, host):
 
             assert host.file(grub2_file).linked_to == linked_to
 
+    @pytest.mark.run_on(['rhel10.0'])
+    @pytest.mark.jira_skip(["CLOUDX-994"])
```

Comment:
Maybe it was not clear enough, but we should skip the test that is related to CLOUDX-994 ticket. Not this test itself. This test will work well on rhel-10 since it's specifically done for rhel-10, so no issues here.

In summary: Remove this marker from this test and move it to `test_hostkey_permissions`.